src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
```

Comment:
Your meta tag and link tag are not correctly formatted. If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.